src/commands.ts

```diff
@@ -1,21 +1,19 @@
-import { Collection, Message, Snowflake } from 'discord.js';
+import { Message } from 'discord.js';
 
 import commandList, { fallback } from './commandHandlers';
 import config from './config';
+import cooldown from './cooldown';
 
 const { COMMAND_PREFIX, defaultEmbed } = config;
 
-const cooldownCollection = new Collection<Snowflake, number>();
 export const commands = async (message: Message) => {
   if (!message.content.startsWith(COMMAND_PREFIX) || message.author.bot) {
     return;
   }
 
-  // handle cooldown
-  const nextCommand = cooldownCollection.get(message.author.id);
-  if(nextCommand && nextCommand > Date.now()) {
-      message.channel.send(`:clock1: Too fast! Only **1** command each **${config.COOLDOWN_SECONDS}** seconds.`);
-      return;
+  if(cooldown.isCooled(message.author.id)) {
```

Comment:
```suggestion
  if (cooldown.isCooled(message.author.id)) {
```

Reply:
oops. When the prettier is implemented that won't happen again 😄